doc/manuals/admin/index.md

```diff
@@ -0,0 +1,67 @@
+# Installing
+
+At the moment, the only supported way of using Quantumleap is through Docker, so there's no actual "installation" of Quantumleap.
+
+If you need to install Docker, refer to [Docker Installation](https://docs.docker.com/engine/installation/).
+
+The Quantumleap Docker Image is hosted at [https://hub.docker.com/r/smartsdk/quantumleap/](https://hub.docker.com/r/smartsdk/quantumleap/).
+
+Now, depending on your scenario, you might have different needs. See from the sections below which fits yours.
+
+
+## All-in-one
```

Comment:
Based on the code i provided for crate, I would provide also how to run in it a Swarm cluster.

Reply:
I think it'd rather add that option in a new PR. That deployment will require a bit more care in the docs :)

Reply:
Just put a placeholder with some coming soon text